warp10/src/main/java/io/warp10/script/functions/BDCLEAR.java

```diff
@@ -0,0 +1,45 @@
+//
+//   Copyright 2024  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.script.functions;
+
+import java.math.BigDecimal;
+
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.WarpScriptStackFunction;
+
+/**
+ * Clear the fractional part
+ */
+public class BDCLEAR extends NamedWarpScriptFunction implements WarpScriptStackFunction {
+
+  public BDCLEAR(String name) {
+    super(name);
+  }
+
+  @Override
+  public Object apply(WarpScriptStack stack) throws WarpScriptException {
+    Object o = stack.pop();
+
+    BigDecimal bd = TOBD.toBigDecimal(getName(), o);
+
+    stack.push(TOBD.toBigDecimal(getName(), bd.toBigInteger()));
+
```

Comment:
This converts a BigDecimal to a BigInteger by stripping the fractional part.

Reply:
It is a bit confusing. 
We already have BDSTRIP, BDCEIL, BDFLOOR, BDROUND... and this one act as a kind of double to long cast. 

What is the function to CLEAR the integer part and return the fractionnal part? 

It could be less confusing to have a BDSPLIT that returns the integer part (so, this function), and the fractional part.